src/main/java/frc/robot/hardware/servo/Servo.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.hardware.servo;
+
+/**
+ * A wrapper class around Servo.
+ * @see edu.wpi.first.wpilibj.Servo
+ */
+public class Servo {
+    private final edu.wpi.first.wpilibj.Servo servo;
+    private boolean inverted = false;
+
+    /**
+     * @param port the PWM port the servo is connected to
+     */
+    public Servo(int port) {
+        servo = new edu.wpi.first.wpilibj.Servo(port);
+    }
+
+    /**
+     * Sets the speed of the servo.
+     * If the servo is inverted, the speed will be set to 1 - speed.
+     * @param speed the speed to set the servo to
+     */
+    public void set(double speed) {
+        if (inverted) {
+            servo.set(1 - speed);
+        } else {
+            servo.set(speed);
+        }
+    }
+
+    /**
+     * Sets the angle of the servo.
+     * @param angle the angle to set the servo to
+     * @see edu.wpi.first.wpilibj.Servo#setAngle(double)
+     */
+    public void setAngle(double angle) {
+        servo.setAngle(angle);
+    }
+
+    /**
+     * @return the position of the servo
+     * @see edu.wpi.first.wpilibj.Servo#get()
+     */
+    public double get() {
+        return servo.get();
+    }
+
+    /**
+     * @return the angle of the servo
+     * @see edu.wpi.first.wpilibj.Servo#getAngle()
+     */
+    public double getAngle() {
+        return servo.getAngle();
+    }
+
+    /**
+     * Sets the servo to be inverted or not.
+     * @param inverted true if the servo should be inverted, false otherwise
+     */
+    public void setInverted(boolean inverted) {
+        this.inverted = inverted;
+    }
+
+    /**
+     * @return the inversion state
+     */
+    public boolean getInverted() {
+        return this.inverted;
+    }
+
+    /**
+     * Toggles the servo enabled status.
+     * @see edu.wpi.first.wpilibj.Servo#setDisabled()
+     */
+    public void toggle() {
+        servo.setDisabled();
```

Comment:
I initially thought this too, but when reading the [documentation](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Servo.html) of the Servo class, the [setDisabled()](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/PWM.html#setDisabled()) method actually toggles the enabled status.

However, the language used is a bit confusing, and I'm not sure if that's what's actually intended.
> Temporarily disables the PWM output. The next set call will re-enable the output.

So I'll probably wait to get a second pair of ears and see what they think. It might be worth changing the name of the function depending on what the setDisabled() function actually does.

Reply:
setDisabled() disables the Servo until the following set() call, per WPILib docs

Remove Inversion, there is no need for that here.